generators/app/index.js

```diff
@@ -40,52 +41,58 @@ module.exports = yeoman.Base.extend({
 
       gems: _(this.gems).concat(
         "gem 'pg', '~> 0.18'",
-        "gem 'puma'",
-        "gem 'rack-cors'"
+        "gem 'puma', '~> 3.0'"
       ).sortBy().join('\n'),
 
       generatorVersion: this.fs.readJSON(require.resolve('../../package.json')).version,
 
-      test_gems: _(this.testGems).concat(
-        "gem 'byebug', platform: :mri"
-      ).sortBy().join('\n  ')
+      testGems: _(this.testGems).concat(
+        "gem 'byebug', platform: :mri",
+        "gem 'dotenv'"
+      ).sortBy().join('\n  '),
+
+      _: _ //lodash
     };
   },
 
   writing: function () {
-    var from = this.templatePath('.');
-    var to = this.destinationPath('.');
-
-    this.fs.copyTpl(from, to, _.merge(this.templateVars, { _: _ }));
-
-    // copyTpl seems to mess with binary files (which is understandable)
-    this.fs.copy(
-      this.templatePath('app/client/assets/foo-asset.png'),
-      this.destinationPath('app/client/assets/foo-asset.png')
-    );
-
-    // See https://github.com/npm/npm/issues/7252 for why this is needed
-    for (let file of fs.readdirSync(this.templatePath())) {
-      if (file.charAt(0) === '_') {
-        this.fs.move(
-          this.destinationPath(file),
-          this.destinationPath(`.${file.substr(1)}`)
-        );
-      }
+    var files = glob.sync('**', {
+      cwd: this.sourceRoot(),
+      dot: true
+    });
+
+    files.forEach(function (file) {
+      this.fs.copyTpl(
+        this.templatePath(file),
+        this.destinationPath(file),
+        this.templateVars
+      );
+    }.bind(this));
+
+    if (this.envFile) {
+      this.fs.write(this.destinationPath('shared/.env'), this.envFile);
+    } else {
+      this.fs.copyTpl(
+        this.templatePath('shared/.env'),
+        this.destinationPath('shared/.env'),
+        this.templateVars
+      );
     }
+
     this.config.save();
   },
 
   install: function() {
-    this.composeWith('prelude:setup', {}, { local: require.resolve('../setup') });
+    this.spawnCommand('chmod', ['+x', 'bin/setup'], { cwd: this.destinationPath() });
+    this.spawnCommand('chmod', ['+x', 'bin/start'], { cwd: this.destinationPath() });
```

Comment:
Ruby on Windows is much better than it was years ago, and it's not like no one is caring.
It is easier for everyone to just switch to BashOnWindows, but there is an infinitude of reasons why people may not want to.

It has some problems, some people believe it may have security issues because of the way it translates the syscalls directly for performance reasons, it is not available on 32-bit Windows, it does not implement everything yet. Right now I can't run nyaovim, I can't use neovim correctly because of terminal issues, I can't run webpack-dev-server, rails or anything that depends on the inotify API.

I also don't think Windows is the best environment for development, that's why I use a virtual machine, but I don't think that just turning our backs to it is the right thing to do.